app/controllers/api/v2x0/service_credential_types_controller.rb

```diff
@@ -0,0 +1,9 @@
+module Api
+  module V2x0
```

Comment:
Since we're following simver and v2.0 is already public, this should be in v2.1

Reply:
Do you mind basing this on v3.0 from https://github.com/RedHatInsights/topological_inventory-api/pull/291 ?